### PR TITLE
#2774 CalculBG named CalculationBG, corrections about blank spaces an…

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
nrt have been checked, they all pass

closes [2774](https://github.com/dynawo/dynawo/issues/2774)

### CODE REVIEW

**Comment by ErwanGuichard49** [comment]
> Thanks. Have you also checked in dynawo-rte and dynawo-itf that there is no "CalculBG" that should be replaced with "CalculationBG" ?

There is no such model in dynawo-rte.

File: nrt/data/SVarC/description/SVarC.tex [hunk 1]
```diff
@@ -155,8 +155,8 @@ \subsection{Step on the reference voltage}
   \begin{tikzpicture}
     \begin{axis}[height = 1.7in]
        \addplot[color=blue!50]
-       table[x=time,y expr=\thisrow{Test_svarc_injector_UPu}*225]
-       {../SVarC_1_StepUref/reference/outputs/curves/curves.csv};
+       table[x=time,y expr=\thisrow{SVarC_SVarC_injector_UPu}*225]
```
Comment (cossartque) [inline nrt/data/SVarC/description/SVarC.tex:158]: Why this change ?

Comment (ErwanGuichard49) [reply nrt/data/SVarC/description/SVarC.tex:158]: The .tex could not be compiled, the variables had not been updated after a previous modification.

**Review by cossartque — APPROVED** [review]
